scripts/validate_container.sh

```diff
@@ -61,6 +61,7 @@ installed_backends=`\
 
 # remote_rest targets are automatically filtered, 
 # so is execution on the photonics backend
+# FIXME - remove items from $available_backends based on how many GPUs are installed?
```

Comment:
No, for one, I think it is good if the examples and tutorial we have don't just crash if you don't have the hardware to run them, but instead give a comprehensive message. Furthermore, it is still good to check that the backends that were primarily intended for multi-gpu usage still work on a single gpu (they should). 

I updated the executing_circuits and the multi-gpu notebooks to check the number of gpus first. I was a bit confused about the multi-gpu notebook, since the cell seemed to never actually wait for the async results, and I am not sure what would be a nice way to give some output beyond the timing.  
```suggestion
```

Reply:
OK, I reverted this placeholder change in lieu of your changes to the notebooks themselves.